resources/assets/less/bem/store-supporter-tag.less

```diff
@@ -0,0 +1,68 @@
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.store-supporter-tag {
+  &__input {
+    .reset-input();
+    .default-border-radius();
+
+    background-color: #eee;
+    color: #555;
+    padding: 4px 8px;
+    text-align: center;
+    min-width: 14em;
+  }
+
+  &__input-feedback {
+    display: inline-block; // prevent collapsing of empty block
+    font-size: 13px;
+    font-weight: 600;
+    color: #555;
+    text-align: center;
+  }
+
+  &__user-icon {
+    margin: 8px;
+  }
+
+  &__user-search {
+    margin: 8px;
+  }
+
+  &--disabled {
```

Comment:
the other benefit is if an element is styled incorrectly, checking the element definition itself is enough instead of figuring out where it's overridden first.